cloud/settings.py

```diff
@@ -43,6 +43,7 @@
     'rest_framework',
     'rest_framework.authtoken',
     'drf_yasg',
+    'uploads',
```

Comment:
Move 'uploads' app below 'accounts', because 'uploads' depends on 'accounts'. This could cause dependency problems in the future